server/config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by toma on 03.11.16..
+ */
+let config = {
+  auth: {
+    password: 'BorzinaITominaAplikacijaZaOodlanje',
```

Comment:
Ok for now, but these usually shouldn't get comitted, but fetched from an environment variable or some other way. No need to do it in this phase.

P.S. Add a TODO denoting this for future reference.